doc/api/dns.md

```diff
@@ -15,8 +15,8 @@ For example, looking up `iana.org`.
 ```js
 const dns = require('dns');
 
-dns.lookup('iana.org', (err, addresses, family) => {
-  console.log('addresses:', addresses);
+dns.lookup('iana.org', (err, address, family) => {
+  console.log('address:', address);
```

Comment:
this PR is fix a mistake, not to enhance the documentation, I can do it but I'm scared that we end up on a very long debate on when to provide examples or not instead of a 5 seconds fix....